core/thread_group.rbs

```diff
@@ -8,7 +8,13 @@
 # Newly created threads belong to the same group as the thread from which they
 # were created.
 #
-class ThreadGroup < Object
+class ThreadGroup
+  # <!-- rdoc-file=thread.c -->
+  # The default ThreadGroup created when Ruby starts; all Threads belong to it by
+  # default.
+  #
+  Default: instance
```

Comment:
Please use `ThreadGroup` instead of `instance`.
(I need to prohibit writing `instance` type in constants, type aliases, and anything it's not clear.)